core/include/moveit/task_constructor/moveit_compat.h

```diff
@@ -57,3 +57,6 @@
 #define MOVEIT_HAS_OBJECT_POSE MOVEIT_VERSION_GE(1, 1, 6)
 
 #define MOVEIT_HAS_STATE_RIGID_PARENT_LINK MOVEIT_VERSION_GE(1, 1, 6)
+
+// moveit_core has MoveItErrorCode class
+#define MOVEIT_CORE_HAS_ERROR_CODE MOVEIT_VERSION_GE(1, 1, 7)
```

Comment:
I have released 1.1.7 for Noetic and we have back-ported #3009 to Melodic as well (and I plan to cut a Melodic release soon as well). Could we simply drop this?